sdk/examples/supplychain_python/sawtooth_supplychain/supplychain_message_factory.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import json
+
+from sawtooth_processor_test.message_factory import MessageFactory
+from sawtooth_supplychain.processor.handler import SUPPLYCHAIN_NAMESPACE
+
+
+class SupplychainMessageFactory(object):
```

Comment:
You could do it as a separate PR, but since the code is new seems like a good time to just fix it.